test/unit/core/network/createNetwork.spec.js

```diff
@@ -19,6 +19,7 @@ describe("createNetwork", () => {
   };
 
   const logger = console;
+  logger.enabled = true;
```

Comment:
Because `logger` is literally `window.console`, this is setting `window.console.enabled` to `true`, which could end up tainting other tests since `window.console` sticks around for other tests. I think at this point we need to make logger not be `window.console`. Instead, let's declare the logger here:

```
let logger;
```

then inside `beforeEach` add the following:

```
logger = jasmine.createSpyObj(logger, ["log"]);
logger.enabled = true;
```

then in the tests we can remove the lines that say:

```
spyOn(logger, "log");
```

because `logger.log` will already be a spy.

Reply:
See new commit for changes, thanks for the recommendations.